python/tests/unit/arcticdb/version_store/test_aggregation.py

```diff
@@ -137,6 +137,60 @@ def test_hypothesis_max_min_agg(lmdb_version_store, df):
     assert_frame_equal(expected, vit.data)
 
 
+@use_of_function_scoped_fixtures_in_hypothesis_checked
+@settings(deadline=None)
+@given(
+    df=data_frames(
+        [
+            column("grouping_column", elements=string_strategy, fill=string_strategy),
+            column("a", elements=non_zero_numeric_type_strategies()),
```

Comment:
We should test aggregating a string column as well, the code paths internally are often quite different.